src/components/ImportJobs/ImportJobs.js

```diff
@@ -56,9 +55,11 @@ class ImportJobs extends Component {
    * @param  {Array<File>} files
    */
   validateFileExtensions(files = []) {
-    const filesType = get(files, [0, 'type'], '');
+    const fileTypeRegex = /\.(\w+)$/;
+    const filesType = files.map(({ name }) => name.match(fileTypeRegex)[1]);
```

Comment:
`filesTypes` name seems to be more appropriate as it is an array of types and not a single type. Please consider renaming.
```suggestion
    const filesTypes = files.map(({ name }) => name.match(fileTypeRegex)[1]);
```